docs/build/configure/_index.md

```diff
@@ -264,6 +264,123 @@ The corresponding raw JSON looks like this:
 
 {{% /expand %}}
 
+## Webhooks
+
+Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud.
+For example, you can configure a webhook to send you a phone notification when your robot's battery drops below 10%.
+
+There is no webhooks tab in the Viam app config builder; use **Raw JSON** mode to configure webhooks as follows:
+
+1. Paste the following JSON template into your raw JSON config.
+   `"webhooks"` is a top-level section like `"components"`, `"services"`, or any of the other config sections.
+
+   {{< tabs >}}
+   {{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+  "webhooks": [
+    {
+      "url": "<Insert your own cloud function or lambda URL for sending the event>",
+      "event": {
+        "attributes": {
+          "data_types": ["binary", "tabular", "file"]
+        },
+        "type": "part_data_ingested"
+      }
+    }
+  ]
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "local",
+      "model": "rdk:builtin:pi",
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": []
+    },
+    {
+      "name": "my_temp_sensor",
+      "model": "rdk:builtin:bme280",
+      "type": "sensor",
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": [],
+      "service_configs": [
+        {
+          "type": "data_manager",
+          "attributes": {
+            "capture_methods": [
+              {
+                "method": "Readings",
+                "additional_params": {},
+                "capture_frequency_hz": 0.017
+              }
+            ]
+          }
+        }
+      ]
+    }
+  ],
+  "webhooks": [
+    {
+      "url": "https://examplecloudfuntionURL-besuretochangethis.net/12345",
+      "event": {
+        "attributes": {
+          "data_types": ["binary", "tabular"]
+        },
+        "type": "part_data_ingested"
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+2. Replace the URL value with the URL of your cloud/lambda function.
+3. Edit the `data_types` list to include only the types of data you want to trigger on.
+4. Configure [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/) for the relevant components.
+   For example, if you want to trigger a webhook on temperature readings, configure data capture and sync on your temperature sensor.
+   Be aware that the component must return the type of data you configured in `data_types`.
+5. Write your lambda function to process the request from `viam-server`.
```

Comment:
Nit: if we support more than just lambda, per `cloud/lambda` usage earlier, let's use that here too:

```suggestion
5. Write your cloud/lambda function to process the request from `viam-server`.
```